core/src/main/scala/org/apache/spark/security/CryptoStreamUtils.scala

```diff
@@ -55,10 +55,8 @@ private[spark] object CryptoStreamUtils extends Logging {
     val params = new CryptoParams(key, sparkConf)
     val iv = createInitializationVector(params.conf)
     os.write(iv)
-    new ErrorHandlingOutputStream(
```

Comment:
While I'm using these crypto streams, I looked at these codes and seems the workaround is not necessary anymore.